packages/datetime2/src/classes.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Classes as DatetimeClasses } from "@blueprintjs/datetime";
+
+const RDP_DAY = "rdp-day";
+const DATEPICKER_NAV_BUTTON = `${DatetimeClasses.DATEPICKER}-nav-button`;
+
+export const Classes = {
+    ...DatetimeClasses,
+    DATEPICKER_DAY: RDP_DAY,
```

Comment:
we probably need to rename these to use "DATEPICKER2" since users might be referencing these class names expecting them to work with DatePicker "v1"